linode_api4/linode_client.py

```diff
@@ -1433,6 +1433,620 @@ def cancel(self):
+                             Content-type header must be sent with the request when
+                             this URL is used, and must be the same as it was when
+                             the signed URL was created.
+                             Required for all methods except “GET” or “DELETE”.
+        :type content_type: str
+
+        :param expires_in: How long this signed URL will be valid for, in seconds.
+                           If omitted, the URL will be valid for 3600 seconds (1 hour). Defaults to 3600.
+        :type expires_in: integer 360..86400
+
+        :param method: The HTTP method allowed to be used with the pre-signed URL.
+        :type method: str
+
+        :param name: The name of the object that will be accessed with the pre-signed
+                     URL. This object need not exist, and no error will be returned
+                     if it doesn’t. This behavior is useful for generating pre-signed
+                     URLs to upload new objects to by setting the method to “PUT”.
+        :type name: str
+
+        :returns: The signed URL to perform the request at.
+        :rtype: str
+        """
+        if method not in ("GET", "DELETE") and content_type is None:
+            raise ValueError(
+                "Content-type header is missing for the current method!"
+            )
+        params = {
+            "method": method,
+            "name": name,
+            "expires_in": expires_in,
+            "content_type": content_type,
+        }
+
+        result = self.client.post(
+            "/object-storage/buckets/{}/{}/object-url".format(
+                cluster_id, bucket
+            ),
+            data=drop_null_keys(params),
+        )
+
+        if "errors" in result:
+            raise UnexpectedResponseError(
+                "Unexpected response when creating the access url of an object!",
+                json=result,
+            )
+
+        return MappedObject(**result)
+
+    def ssl_cert_delete(self, cluster_id, bucket):
+        """
+        Deletes this Object Storage bucket’s user uploaded TLS/SSL certificate
+        and private key.
+
+        API Documentation: https://www.linode.com/docs/api/object-storage/#object-storage-tlsssl-cert-delete
+
+        :param cluster_id: The ID of the cluster this bucket exists in.
+        :type cluster_id: str
+
+        :param bucket: The bucket name.
+        :type bucket: str
+        """
+
+        resp = self.client.delete(
+            "/object-storage/buckets/{}/{}/ssl".format(cluster_id, bucket)
+        )
+
+        if "errors" in resp:
+            raise UnexpectedResponseError(
+                "Unexpected response when deleting a bucket!",
+                json=resp,
+            )
+        return True
+
+    def ssl_cert(self, cluster_id, bucket):
+        """
+        Returns a boolean value indicating if this bucket has a corresponding
+        TLS/SSL certificate that was uploaded by an Account user.
```

Comment:
This method doesn't seems to return boolean...

Reply:
The boolean value is wrapped in the result object. The description is for the endpoint response itself. Same for the next method.